src/ReactionService/appsettings.json

```diff
@@ -0,0 +1,48 @@
+{
+  "Serilog": {
+    "Using": [],
+    "LevelSwitches": { "$controlSwitch": "Information" },
+    "MinimumLevel": { "ControlledBy": "$controlSwitch" },
+    "Enrich": [
+      "FromLogContext",
+      "WithMachineName",
+      "WithProcessId",
+      "WithThreadId"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Console"
+      },
+      {
+        "Name": "Seq",
+        "Args": {
+          "serverUrl": "http://localhost:9864",
```

Comment:
"serverUrl": "http://localhost:5341"

it's not ReactionService localhost, it's Seq

Reply:
Done